tools/github/requirements.txt

```diff
@@ -1,3 +1,233 @@
-PyGithub==1.55 \
```

Comment:
im not clear if this is required for anything - afaict nothing was actually installing from this file